src/lib/imgio.cc

```diff
@@ -4,7 +4,6 @@
 #include <cstdlib>
 #include <vector>
 #define cimg_display 0
-#define cimg_use_jpeg
```

Comment:
Most users would definitely need jpeg support. I think we should keep it enabled by default.